Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift

```diff
@@ -2,178 +2,185 @@
 // Copyright © 2020 Stream.io Inc. All rights reserved.
 //
 
-import Combine
-import StreamChatClient
-import SwiftUI
+#if swift(>=5.3)
 
-@available(iOS 14, *)
-struct ChatView: View {
-    /// The `ChannelController` used to interact with this channel. Will be synchronized in `onAppear`.
-    @StateObject var channel: ChatChannelController.ObservableObject
-    /// The `text` written in the message composer.
-    @State var text: String = ""
-    /// Binding for message actions and channel actions ActionSheet.
-    @State var actionSheetTrigger: Bool = false
-    /// Message being edited.
-    @State var editingMessage: ChatMessage?
+    import Combine
+    import StreamChatClient
+    import SwiftUI
```

Comment:
nit: I don't think we need to intend code wrapped inside `#if endif`

Reply:
hmm, seems like formatter indented it, not me.  I agree with you, let me check if I can disable that

Reply:
> Question: should we also build sample app in iOS12?
> I'd say yes just because we have so many `#if` checks in there :)

Fixed in 5e88f6bb42f75923f4910d6c27fd271c82fa0f08

Reply:
Fixed in f2b08921dda0f6e3d7f7481d91bfd4db3aeb88b3